server/tinylicious/package.json

```diff
@@ -28,18 +28,18 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "@fluidframework/common-utils": "^0.26.0",
-    "@fluidframework/gitresources": "^0.1017.1",
-    "@fluidframework/protocol-base": "^0.1017.1",
-    "@fluidframework/protocol-definitions": "^0.1017.1",
+    "@fluidframework/common-utils": "^0.29.0",
+    "@fluidframework/gitresources": "^0.1022.0",
```

Comment:
it doesn't look like tinylicious has been updated for a while, any reason we need to do it this round? i see the build is failing, and looks like server breaking changes

Reply:
i would just revert the tinylicious change, and file a bug

Reply:
I'm not sure, this was just the result of running the bump command

Reply:
Okay I'm going to try manually reverting the package.json changes in tinylicious, then `npm i`